app/styles/ember-ebau-gwr.scss

```diff
@@ -29,6 +30,14 @@
         min-width: calc(100% - 2 * #{$modal-padding-horizontal-m});
       }
     }
+
+    div.uk-modal .uk-modal-body .ember-power-select-status-icon {
+      @extend .uk-invisible;
+    }
+
+    .ember-ebau-gwr-modal-login {
+      height: 400px;
+    }
```

Comment:
What is this custom css for?

Reply:
The municipality dropdown only shows within the modal, otherwise it's a scrollable container. If the modal is too small, the visible area of the dropdown is very small. It's only a temporary solution, since the login dialog will be reworked.